modtools/create-unit.lua

```diff
@@ -338,13 +340,6 @@ function domesticate(uid, group_id)
     u.animal.population.population_idx = -1
     u.animal.population.depth = -1
 
-    u.counters.soldier_mood_countdown = -1
```

Comment:
Which change is this related to? This section was last changed in 96b118b2e7fe8d91b59e0cc1c7951bf27a5e6698, which led to the merge conflict here, but I can't imagine the first two fields here (at least) corresponding to the bugs you fixed.

Reply:
And for reference, the rest of this section dates back to the [initial commit](https://github.com/DFHack/scripts/blame/038499117c6391431f2e2d57d73856254029c391/modtools/create-unit.lua#L341) of this script (only the field names were changed recently)

Reply:
To clarify, I did not remove this section, but simply relocated it from the domesticate() function to line 552. This is because, as far as I can tell, these changes should apply to all units, not just tamed creatures.

Reply:
Ah, thanks